### PR TITLE
28 add joplin building functionality recovery time model

### PR DESCRIPTION
**Pull request by ncsa-mo**
The test script includes the restoration dataset

### CODE REVIEW

**Comment by ncsa-mo** [comment]
close #28

File: pyincore/analyses/joplinempiricalrestoration/joplinempiricalrestoration.py [hunk 1]
```diff
@@ -50,6 +56,15 @@ def run(self):
 
         building_func = building_func_5[["guid", "LS_0", "LS_1", "LS_2", "haz_expose"]].copy()
         building_func["targetFL"] = target_fl
+        if building_target_fl is not None:
+            building_func = pd.merge(building_func, building_target_fl,
+                                         how="left", on="guid", copy=True, validate="1:1")
```
Comment (navarroc) [inline pyincore/analyses/joplinempiricalrestoration/joplinempiricalrestoration.py:61]: I get a pep8 warning about this line

File: pyincore/analyses/joplinempiricalrestoration/joplinempiricalrestoration.py [hunk 2]
```diff
@@ -50,6 +56,15 @@ def run(self):
...
+            # Replace NaN value from targetFL_y with targetFL_x value
+            building_func["targetFL"] = building_func["targetFL_y"].fillna(building_func["targetFL_x"])
+            # Drop unwanted columns
+            building_func = building_func.drop(["targetFL_x", "targetFL_y"], axis=1)
+            building_func = building_func.astype({"targetFL": "int64"})
+
```
Comment (navarroc) [inline pyincore/analyses/joplinempiricalrestoration/joplinempiricalrestoration.py:68]: extra blank line

**Comment by navarroc** [comment]
Line 85 of joplinempiricalrestoration.py refers to "building_dmg", should be building_funct

**Comment by navarroc** [comment]
The code looks good, just a few minor pep8 and documentation changes that need to be addressed.

**Pull request closed without merging**